packages/compiler-cli/src/ngcc/BUILD.bazel

```diff
@@ -4,23 +4,14 @@ load("//tools:defaults.bzl", "ts_library")
 load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test")
 
 ts_library(
-    name = "test_lib",
-    testonly = 1,
+    name = "ngcc",
     srcs = glob([
+        "*.ts",
```

Comment:
Is this necessary? Doesn't `**/*.ts` match top-level files as well?

Reply:
I don't think so. Perhaps it ensures that the top level source files are included first?